### PR TITLE
Tasks Redesign: Colors, Title, Notes, Habit Controls, Button Styles 

### PR DESCRIPTION
**Pull request by negue**
This PR extracts the following changes from #12142 
- [x] New Task Colors
- [x] Title / Notes in the Modal
- [x] New Habit-Controls 
- [x] Delete Button (Modal)
- [x] Save / Cancel
- [x] Footer
- [x] Hide Advanced Settings when creating a Task

### CODE REVIEW

**Comment by Tressley** [comment]
These LGTM. 👍

**Comment by negue** [comment]
I'm adding the Save / Cancel styling from #12142 too and then this can be reviewed / merged.

File: website/client/src/store/getters/tasks.js [hunk 1]
```diff
@@ -13,6 +13,15 @@ export function getTagsFor (store) {
     .map(tag => tag.name);
 }
 
+export function getTagsByIdList (store) {
```
Comment (paglias) [inline website/client/src/store/getters/tasks.js]: is this used somewhere? i can't find it

**Review by paglias — COMMENTED** [review]
Left a comment about a small piece of JS but mostly these are CSS changes so I'll let @Tressley merge it

File: website/client/src/store/getters/tasks.js (hunk 1 above)
Comment (negue) [reply website/client/src/store/getters/tasks.js]: oh this is a part of the tags-refactoring (in #12142) - gonna remove it here

**Comment by Tressley** [comment]
Merge it!

**Pull request merged**